hack/generate/changelog/util/fragment.go

```diff
@@ -176,7 +180,8 @@ func (g *gitPullRequestNumberGetter) GetPullRequestNumberFor(filename string) (u
 }
 
 func (g *gitPullRequestNumberGetter) getCommitMessage(filename string) (string, error) {
-	args := fmt.Sprintf("log --follow --pretty=format:%%s --diff-filter=A --find-renames=40%% %s", filename)
+	//nolint:lll
+	args := fmt.Sprintf("log --follow --pretty=format:%%s --diff-filter=A --find-renames=%s%% %s", similarityIndex, filename)
```

Comment:
here we can always break the line. 
The lint exists because it hard to understand/read the code when the line is too long.

Reply:
Also, since `similarityIndex` it is not used outside of `getCommitMessage ` do not need to be a global var. I mean can be used inside of the func;
```suggestion
	args := fmt.Sprintf("log --follow --pretty=format:%%s --diff-filter=A --find-renames=%s%% %s", "90", filename)
```

Reply:
I'm fine with keeping this as a constant since it's easier to see, but perhaps move it above the consuming method.

Reply:
@rashmigottipati @camilamacedo86 WDYT of bumping up the `lll` linter to allow slightly longer lines? IMO, the split of this line actually makes it _less_ readable.

No need to change this line for now. This is just something to think about and follow-up on later.

Reply:
Personally I’d like to get rid of `lll` altogether. It’s more of a nuisance than a value add.